### PR TITLE
Implement Company model

### PR DESCRIPTION
**Pull request by mroetheli**

### CODE REVIEW

**Comment by mroetheli** [comment]
Für die neuen models wurden erst model tests geschrieben, die restlichen tests werden zusammen mit der CRUD-Umsetzung geschrieben.

File: app/controllers/companies_controller.rb [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  end
+
+  private
+    # Use callbacks to share common setup or constraints between actions.
+    def set_company
+      @company = Company.find(params[:id])
+    end
+
+    # Only allow a trusted parameter "white list" through
+# rubocop:disable LineLength
```
Comment (psunix) [inline app/controllers/companies_controller.rb]: don't ignore this rule :)
just put the arguments on multiple lines.

File: db/migrate/20180316095930_create_companies.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      t.string :email
+      t.string :phone
+      t.string :partnermanager
+      t.string :contact_person
+      t.string :email_contact_person
+      t.string :phone_contact_person
+      t.string :crm
+      t.string :level
+      t.string :picture
+      t.boolean :my_company
```
Comment (psunix) [inline db/migrate/20180316095930_create_companies.rb]: always use default value for boolean attributes
in this case probably false
t.boolean :my_company, default: false

File: db/schema.rb [hunk 3]
```diff
@@ -70,6 +66,15 @@
     t.index ["person_id"], name: "index_educations_on_person_id"
   end
 
+  create_table "employee_quantities", force: :cascade do |t|
+    t.string "category"
+    t.integer "quantity"
+    t.bigint "company_id"
+    t.datetime "created_at", null: false
```
Comment (psunix) [inline db/schema.rb]: isn't there t.timestamps ?

Comment (mroetheli) [reply db/schema.rb]: should this be generated automatically?

Comment (psunix) [reply db/schema.rb]: yes, the schema.rb is for example updated when you run rake db:migrate
make sure the newest version of schema.rb is checked in.

Comment (mroetheli) [reply db/schema.rb]: In the schema.rb all entities have a created_at and updated_at timestamp with datatype t.datetime, there are no t.timestamps. I'm not quite sure, what is false?

Comment (psunix) [reply db/schema.rb]: there's nothing wrong ... it's just rails default to use t.timestamps
http://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/Table.html#method-i-timestamps
you then have only one line :)

**Pull request merged**